src/main/java/org/zalando/nakadi/controller/EventTypeController.java

```diff
@@ -145,7 +154,11 @@ public EventTypeController(final EventTypeService eventTypeService,
         }
 
         eventTypeService.update(name, eventType);
-        return status(HttpStatus.OK).build();
+        final HttpHeaders headers = new HttpHeaders();
+        if (!nakadiSettings.getWarnAllDataAccessMessage().isEmpty()) {
+            headers.add(HttpHeaders.WARNING, nakadiSettings.getWarnAllDataAccessMessage());
```

Comment:
According to the standard, the warning header format should contain code and app name.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Warning
It can be added here or in senza files but I don't see it there either.

Reply:
I saw that page, but it seems that this format is specifically for cache-related warnings, isn't it? Do you think it applies here as well?

Reply:
No, we should use 299 code "Miscellaneous Warning | Same as 199(Arbitrary, non-specific warning), but indicating a persistent warning"
 standard very specific about it https://tools.ietf.org/html/rfc7234#section-5.5
```
Warnings can be used for other purposes, both cache-related and
   otherwise.  The use of a warning, rather than an error status code,
   distinguishes these responses from true failures.
```

Reply:
@SergKam dc56340